jsonxx.h

```diff
@@ -544,7 +562,11 @@ number_value_ = static_cast<long double>(n); \
         *this << Value(value);
         return *this;
     }
-    
+
+    static const Value EmptyValue = {};
+
+    static const Boolean True = true;
+    static const Boolean False = false;
```

Comment:
Unless clang-format sent them here, can you move these definitions to the top with the rest?

Reply:
Boolean can be moved up. Value needs to be after the class definition, but it can be higher up a bit.